cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -1584,13 +1597,16 @@ function Configure_Containerd {
     replace('CNI_CONF_DIR', ${env:CNI_CONFIG_DIR})
 }
 
-# Register and start containerd service.
+# Register if needed and start containerd service.
 function Start_Containerd {
-  Log-Output "Creating containerd service"
-  & containerd.exe --register-service --log-file ${env:LOGS_DIR}/containerd.log
+  # Do the registration only if the containerd service does not exist.
+  if (Get-WMIObject -Class Win32_Service -Filter  "Name='containerd'" -eq $null) {
+    Log-Output "Creating containerd service"
+    & containerd.exe --register-service --log-file ${env:LOGS_DIR}/containerd.log
```

Comment:
Should the path be quoted?

Reply:
It have been already as is, and it'll work except if the LOGS_DIR has spaces or any special chars. Will definitely have it quoted, thanks for the catch!